A3A/addons/scrt/Encounter/fn_encounter_fleeingCivConvoy.sqf

```diff
@@ -95,17 +95,19 @@ private _fnc_fillcargo = {
 	params ["_vehObj","_civVehicles"];
 	private _vehdriver = driver _vehObj;
 	private _vehObjgroup = group _vehdriver;
-    if (civTraffic isEqualTo 2) then {
+    if (civTraffic isEqualTo 2 || civTraffic isEqualTo 4) then {
 	    _cargoSpace = [_civVehicles, true] call BIS_fnc_crewCount;
         _cargoSpace = _cargoSpace - 1;
 	    //_vehObj setVehicleLock "LOCKED";
 	    /* while {_cargoSpace != 1} do { ///driver already exists
             
         }; */
-        for _i from 1 to _cargoSpace do {
+        for "_i" from 1 to _cargoSpace do {
             _cargo = [_vehObjgroup, FactionGet(civ,"unitMan"), getPos _vehObj, [], 10] call A3A_fnc_createUnit;
-            _cargo moveInAny _vehObj; 
-            _cargo disableAI "MOVE";
+            _cargo moveInAny _vehObj; ///stupid fucks won't sit in a vehicle, it's not fixable IT'S NOT POSSIBLE, FUCK ARMA AI
+            _cargo disableAI "MOVE"; ///if you assign them to vehicle, it will only get worse
+            _cargo disableAI "PATH"; ///they will jump out immediately, CUNTS
+            _cargo disableAI "FSM";  /// :-.
```

Comment:
I suggest sleeping on it, think of something else... and if that fails, delete this entire `for` block

Reply:
```
private _fnc_fillcargo = {
    params ["_vehObj", "_civVehicles"];
    private _vehdriver = driver _vehObj;
    private _vehObjgroup = group _vehdriver;

    if (civTraffic isEqualTo 2 || civTraffic isEqualTo 4) then {
        
        _cargoSpace = [_civVehicles, true] call BIS_fnc_crewCount;
        _cargoSpace = _cargoSpace - 1;

        for "_i" from 1 to _cargoSpace do {
            private _cargo = [_vehObjgroup, FactionGet(civ, "unitMan"), getPos _vehObj, [], 10] call A3A_fnc_createUnit;

            _cargo assignAsCargoIndex [_vehObj, _i];  
            _cargo disableAI "MOVE";

            sleep 0.2;

            _cargo moveInCargo [_vehObj, _i];
            group _cargo setVariable ["deleteGroupWhenEmpty", true];
        };
    };
};
```
Possible solution, though haven't tried that out.

Reply:
Nope